src/main/frontend/modules/instrumentation/sentry.cljs

```diff
@@ -17,4 +17,4 @@
 (defn init []
   (let [config (clj->js config)
         init-fn (if (util/electron?) Sentry-electron/init Sentry/init)]
-    (Sentry/init config)))
+    (init-fn config)))
```

Comment:
Looks like we weren't using the right library for electron. I could QA this if I get an account

Reply:
Good catch! I just sent you a sentry invite! Lmk if you need anything else.

Reply:
Thanks! I did not test this yet. I can test this tomorrow